plugin.video.iplayerwww/media/LICENSE.txt

```diff
@@ -0,0 +1,6 @@
+Icons made by:
```

Comment:
in the future consider to move these images to a resource based addon and add it as a dependency, this way other addons can use them.

Reply:
Good idea, this reduces the download volume for updates. Not for this release, though.